packages/at_secondary_server/lib/src/verb/handler/sync_progressive_verb_handler.dart

```diff
@@ -43,7 +43,8 @@ class SyncProgressiveVerbHandler extends AbstractVerbHandler {
     // Get entries to sync
     var commitEntryIterator = atCommitLog!.getEntries(
         int.parse(verbParams[AtConstants.fromCommitSequence]!) + 1,
-        regex: verbParams['regex']);
+        regex: verbParams['regex'],
+        skipDeletes: verbParams['skipDeletes'] == 'true');
```

Comment:
VerbSyntax also needs this parameter.

Reply:
yes. those changes are in at_commons
https://github.com/atsign-foundation/at_libraries/pull/696/files